alignak/daemon.py

```diff
@@ -198,6 +199,8 @@ def __init__(self, name, config_file, is_daemon, do_replace, debug, debug_file):
         self.debug = debug
         self.debug_file = debug_file
         self.interrupted = False
+        self.pidfile = None
+        self.http_interface = GenericInterface(object)
```

Comment:
No because in the code, we use properties or functions of GenericInterface, so I initialize it with that